assets/javascripts/initializers/communitarian.js.es6

```diff
@@ -1,13 +1,36 @@
 import { withPluginApi } from "discourse/lib/plugin-api";
+import discourseComputed from "discourse-common/utils/decorators";
+
+const isHomePageField = {
+  isHomePage: window.location.pathname === "/",
```

Comment:
When is this value evaluated? Every time I visit some page?

Reply:
Depends on whether the corresponding route is called. If, as in one of our cases, the `discovery-categories` route is called, then this field will be declared in the `discovery/categories` controller and it will be available in the `discovery/categories` template. In our case, these are now the "/" and "/categories" pages. After allocating a separate route for the home page, some of this may change.